sdk/core/Azure.Core.Experimental/api/Azure.Core.Experimental.net461.cs

```diff
@@ -120,3 +120,73 @@ public ProvisionableTemplateAttribute(string resourceName) { }
         public string ResourceName { get { throw null; } }
     }
 }
+namespace System.ClientModel
+{
+    public partial interface IAuthorizationCodeFlowContext : System.ClientModel.IScopedFlowContext, System.ClientModel.ITokenContext
+    {
+        System.Uri AuthorizationUri { get; }
+        System.Uri RefreshUri { get; }
+        System.Uri TokenUri { get; }
+    }
+    public partial interface IClientCredentialsFlowContext : System.ClientModel.IScopedFlowContext, System.ClientModel.ITokenContext
```

Comment:
Do we need all these flow types public? There seems to be so much surface area in this PR.

Reply:
These are the flow types that exist in the spec, so it should be possible to implement any of them.